### PR TITLE
Doc typo

### PR DESCRIPTION
**Pull request by sciunto**
@emmanuelle This is a doc fix. The title didn't match the picture content.

I don't think it's important to show the mask. Instead, I prefer to show the original and the defected images.

### CODE REVIEW

File: doc/examples/filters/plot_inpaint.py [hunk 1]
```diff
@@ -43,12 +43,12 @@
 
 fig, axes = plt.subplots(ncols=3, nrows=1)
 
-axes[0].set_title('Defected image')
+axes[0].set_title('Original image')
 axes[0].imshow(image_orig)
 axes[0].set_xticks([]), axes[0].set_yticks([])
 
-axes[1].set_title('Defect mask')
-axes[1].imshow(mask, cmap=plt.cm.gray)
```
Comment (soupault) [inline doc/examples/filters/plot_inpaint.py]: I found it more representative to show defect mask, not defected image.
We might consider adding `defected image` between `defected mask` and `inpainted image`, so creating 2x2 matrix of images.

**Comment by soupault** [comment]
@sciunto thanks for reporting! Now, when we see how the example is being rendered, may I ask you to add also `plt.tight_layout()` before `plt.show()`?

**Comment by sciunto** [comment]
@soupault I made the additional changes. Please, let me know if it doesn't look good.

**Comment by soupault** [comment]
@sciunto perfect, thanks!
I suppose you might be asked by the core devs to squash stack of commits into single commit, but I have no idea how to do that.

**Comment by sciunto** [comment]
Yes, i'll squash the commits. I fixed a typo in another example as well.

**Comment by soupault** [comment]
@sciunto good! :+1: from me, waiting for one more :).

File: doc/examples/filters/plot_inpaint.py [hunk 2]
```diff
@@ -41,18 +41,24 @@
...
-fig, axes = plt.subplots(ncols=3, nrows=1)
+fig, axes = plt.subplots(ncols=2, nrows=2)
+ax0, ax1, ax2, ax3 = axes.ravel()
 
-axes[0].set_title('Defected image')
-axes[0].imshow(image_orig)
-axes[0].set_xticks([]), axes[0].set_yticks([])
+ax0.set_title('Original image')
+ax0.imshow(image_orig)
+ax0.set_xticks([]), ax0.set_yticks([])
```
Comment (emmanuelle) [inline doc/examples/filters/plot_inpaint.py]: ``ax0.axis('off')`` for less boilerplate code

**Comment by emmanuelle** [comment]
I suggested a stylistic improvement.

I also think this example would be better if a smaller picture was used (such as a zoom on a part of the astronaut image), because it's hard to see the effect of the inpainting on the image as it is now.

**Comment by sciunto** [comment]
@emmanuelle I made your suggestion, Thanks. I have some problems with cython in my virtualenv, so it's not possible for me to check a more complex modification for now. I'll need to figure out how to fix that. I didn't try to reduce the size of the picture, but it's also an excellent suggestion, I had the same feeling.

**Comment by emmanuelle** [comment]
Thanks @sciunto We'll reduce the size of the picture in another time, would you mind opening an issue for the record? 

So, here is the second :+1: , therefore I'm merging.

**Comment by sciunto** [comment]
Done :) I'll try to fix my virtualenv this week. It's frustrating.

**Pull request merged**